neqo-transport/src/recv_stream.rs

```diff
@@ -120,9 +132,10 @@ impl RecvStreams {
 /// from incoming STREAM frames.
 #[derive(Debug, Default)]
 pub struct RxStreamOrderer {
+    // TODO: Consider shrinking from time to time?
```

Comment:
Maybe open an issue to track rather than a comment.

Reply:
With the above discussion (https://github.com/mozilla/neqo/pull/1868#discussion_r1900086913) in mind, I removed the TODO. In case it ever shows up in memory profiles, we can reconsider periodic shrinking.